utils/language.py

```diff
@@ -0,0 +1,19 @@
+class Language:
+    def language_switcher(self, required_language, str):
```

Comment:
Good approach to abstract language 👍 
This allows switching language at run time ...

Reply:
Thanks Sir, Venu Sir explained it that we should use more Single Responsibility Principle during our class.